### PR TITLE
Fixing some Mongolian

### LINKED ISSUE #38916: Missing translations for Mongolian (mn)
Hello,

There are some missing translation for Mongolian (mn). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38916
| License       | MIT
| Doc PR        |

Hey @luffy1727, could you help me with the last two strings here?

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf [hunk 1]
```diff
@@ -131,20 +131,20 @@
...
             </trans-unit>
-            <trans-unit id="39">
+            <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Энэ утга үнэн зөв хэл биш байна .</target>
+                <target>This value is not a valid language.</target>
+            </trans-unit>
+            <trans-unit id="39">
+                <source>This value is not a valid locale.</source>
+                <target>This value is not a valid locale.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf]: This needs to be translated

File: src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf [hunk 2]
```diff
@@ -131,20 +131,20 @@
...
-            <trans-unit id="38">
+            <trans-unit id="37">
                 <source>This is not a valid IP address.</source>
                 <target>IP хаяг зөв биш байна.</target>
             </trans-unit>
-            <trans-unit id="39">
+            <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Энэ утга үнэн зөв хэл биш байна .</target>
+                <target>This value is not a valid language.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf]: This needs to be translated

File: src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf (hunk 1 above)
Comment (luffy1727) [reply src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf]: you could say "Энэ утга үнэн зөв байршил биш байна"

File: src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf [hunk 3]
```diff
@@ -131,20 +131,20 @@
...
             </trans-unit>
-            <trans-unit id="38">
+            <trans-unit id="37">
                 <source>This is not a valid IP address.</source>
                 <target>IP хаяг зөв биш байна.</target>
             </trans-unit>
-            <trans-unit id="39">
+            <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Энэ утга үнэн зөв хэл биш байна .</target>
```
Comment (luffy1727) [inline src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf:147]: this was actually correct

Comment (Nyholm) [reply src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf:147]: Google translated translated this to "This meaning is not accurate". 

Can you (once again) confirm that it is correct?

File: src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf (hunk 1 above)
Comment (Nyholm) [reply src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf]: Make sure it refers to a locale, like "sv", "en", "fr", "mn". Google translates this to "location". 

Is there a better (maybe more technical) word we can use?

**Comment by nicolas-grekas** [comment]
Thank you @Nyholm.

File: src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf (hunk 1 above)
Comment (luffy1727) [reply src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf]: yeah i don`t think there is. The closest thing would be "Энэ утга үнэн зөв орчин биш байна"

**Pull request merged**